backend/pyproject.toml

```diff
@@ -65,5 +60,8 @@ module = "celery.*"
 ignore_missing_imports = true
 
 [build-system]
-requires = ["poetry-core", "poetry-dynamic-versioning"]
-build-backend = "poetry_dynamic_versioning.backend"
+requires = ["hatchling", "uv-dynamic-versioning"]
```

Comment:
This fixes the versioning visible to the .../version endpoint, which was broken up until now(the backend API server has been reporting to the charm an unchanged 0.0.0).